src/brs/http/API.java

```diff
@@ -145,11 +150,20 @@ public API(TransactionProcessor transactionProcessor,
         dosFilterHolder.setAsyncSupported(true);
       }
 
-      apiHandler.addServlet(new ServletHolder(new APITestServlet(apiServlet)), "/test");
+      apiHandler.addServlet(new ServletHolder(new APITestServlet(apiServlet)), apiTestPath);
+
+      RewriteHandler rewriteHandler = new RewriteHandler();
+      rewriteHandler.setRewriteRequestURI(true);
+      rewriteHandler.setRewritePathInfo(false);
+      rewriteHandler.setOriginalPathAttribute("requestedPath");
+      rewriteHandler.setHandler(apiHandler);
+      Rule rewriteToRoot = new RewriteRegexRule("^(?!"+regexpEscapeUrl(apiPath)+"|"+regexpEscapeUrl(apiTestPath)+").*$", "/index.html");
```

Comment:
I forgot to ask, is this backwards-compatible with the old jquery ui?

Reply:
Yep